### ISSUE TITLE
Import receives zero matches on Comic Vine

### ISSUE THREAD

**Issue by tigerkzr** [mylar3/mylar3#1365]
**Describe the bug**
I am trying to import comics from a download directory.  Watching the debug log, every file receives 0 for search results when the import goes looking for Comic Vine info. 

**CarePackage**

[carepackage_beedfc75c4595a7f110093e908e4af0403656cb8_(python3-dev).zip](https://github.com/mylar3/mylar3/files/11684366/carepackage_beedfc75c4595a7f110093e908e4af0403656cb8_.python3-dev.zip)

**Environment (please complete the following information):**
 - OS: docker linuxserver/mylar3:nightly running on ubuntu 23.04 server
 - Branch / commit: python3-dev/[beedfc75c4595a7f110093e908e4af0403656cb8]

**Additional information**
Add any other information about the problem here.

**Comment by evilhero** [comment]
It looks like you haven't done anything from the import results management screen - it just scanned in the files and indicated that it had completed scanning successfully. Once the scan is completed and it shows the breakdown of what it found numerically, an import results management button will appear in the lower right that will bring you to page to select what you want to import. If it doesn't appear for some reason, you can also get to it by going into the ``Manage`` tab / ``Advanced Options`` / ``Import Results Management``. 

I do see lots of nzb/torrent provider searches being done for wanted items, which is completely different than what the import does. 

The vast majority of your files that you are trying to import don't contain metadata from what I could tell, and without metadata it can only parse the filename for the data. That being said, most of the filenames I saw in the log also didn't have enough info in the filename to be able to parse anything from  (``IM01.cbr``), which will be a problem if the importer tries to compare names against to CV for valid matches.

**Comment by tigerkzr** [comment]
the log items are from clicking on "import" on the Import results page you mention. The filename contains the name of the comic and the issue number but still returns zero results from comic vine.

```
2023-06-09 23:47:19 | INFO | [IMPORT] Import completed.
-- | -- | --
2023-06-09 23:47:19 | INFO | [IMPORT] Could not find any matching results against CV. Check the logs and perhaps rename the attempted file(s)
2023-06-09 23:47:14 | INFO | [IMPORT][001 The Uncanny X-Men] Now preparing to import. First I need to determine the highest issue, and possible year(s) of the series.
2023-06-09 23:45:43 | INFO | Successfully scanned in directory. Enabling the importResults button now.
```

How much does it need to match on CV? At a min "the uncanny x-men" should return at least a few results.

**Comment by evilhero** [comment]
Yeah, unfortunately such is the way of naming in comics where there is no set standard and its literally the wild west for naming conventions.

I can tell you that having the issue number in the first position is generally how story arc reading orders work - the digits at the very beginning of the filename pertain to the reading order followed by the series name and then issue number, etc. So if mylar sees a numeric there it will assume it's the reading order number, as I think CT might be doing as well. 

For mylar to be able to generally parse items that don't contain any metadata within the file (or are cbr), the format we try to tell everyone is to try and keep simple - ``Seriesname issue (year) ``,  where adding the volume after Seriesname (ie. ``v3 or ``v2012``) will help even more with auto detection where multiple volumes exist of a series.

The directory structure isn't used by Mylar outside of keeping with the 1 series per folder motto (which can be changed on the import start screen), but it won't impact where it gathers the information from.

If you want help or to talk to others on how they have theirs setup, our discord is loaded with users with varying setups and might help you if you're stuck at all.

**Comment by tigerkzr** [comment]
Thanks for the advice!  

Pretty sure the primary issue is that I assumed the files I received had already been part of ordered collection so "naturally they would have been put in a proper directory structure before me". Live and learn, this is my first foray into the digital comics world so I can accept a bit of a learning curve. 

Thanks again! Will check on discord first next time I hit a snag.
